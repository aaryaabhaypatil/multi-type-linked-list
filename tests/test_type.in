NEW 2
 
aa
NEW 1
aarya
TYPE 0
TYPE 1
TYPE 2
