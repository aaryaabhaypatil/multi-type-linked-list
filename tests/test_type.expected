List 0:   -> aa
List 1: aarya
char -> string
string
